### PR TITLE
fix: icon looks clickable in timeline

### PR DESCRIPTION
**Pull request by gitstart**
### Description
Fixes/Partially fixes [Parabol Issue 7016](https://github.com/ParabolInc/parabol/issues/7016)

### Demo
[Loom Video](https://www.loom.com/share/9f0b90ad6aed425990037784e34452ed)

### Testing Scenarios
[Please list all the testing scenarios a reviewer has to check before approving the PR]

- Icon looks less clickable to prompt user to click on the "Invite Teammate" button.
- Upon hovering over Icon, it becomes more visible with a pointer cursor indicating "clickability".
- Upon Clicking, Icon should have the same functionality as "Invite Teammate" button.

### Final Checklist

- [x] I checked the [code review guidelines](https://github.com/GitStartHQ/client-parabol/docs/codeReview.md)    
- [x] I have added [Metrics Representative](https://github.com/GitStartHQ/client-parabol/docs/codeReview.md#metrics-representative) as reviewer(s) if my PR involves metrics/data/analytics related changes.
- [x]  I have performed a self-review of my code, the same way I'd do it for any other team member.
- [x] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions.
- [x] Whenever I took a non-obvious choice I added a comment explaining why I did it this way.
- [x] I added the label One Review Required if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient.
- [x] PR title is human readable and could be used in changelog.

### CODE REVIEW

File: packages/client/components/SuggestedActionCard.tsx [hunk 1]
```diff
@@ -46,14 +47,19 @@ const CancelIcon = styled(Icon)({
...
   background: PALETTE.SLATE_300,
   borderRadius: '100%',
   boxShadow: buttonShadow,
   padding: 8,
   position: 'absolute',
   fontSize: ICON_SIZE.MD36,
   top: 100,
-  userSelect: 'none'
+  userSelect: 'none',
+  opacity: 0.7,
```
Comment (Dschoordsch) [inline packages/client/components/SuggestedActionCard.tsx:59]: -2 this makes the problem only worse. `SuggestedActionCard` is used in multiple places and now looks clickable in all of those regardless of the icon used.

@ackernaut The issue was not well defined what the best solution would be. I don't mind having the icon repeat the primary action of a suggestion, but maybe there are some guideline against this?

**Review by Dschoordsch — CHANGES_REQUESTED** [review]
This needs some clarification first if making the icon clickable (in which case it should be clickable in all suggested actions) or if we change the visual to not look like a button.

File: packages/client/components/SuggestedActionCard.tsx (hunk 1 above)
Comment (gitstart) [reply packages/client/components/SuggestedActionCard.tsx:59]: @Dschoordsch  thanks for the feedback. You can let me know what you decide

**Comment by Dschoordsch** [comment]
@ackernaut could you check the comments above?

**Comment by ackernaut** [comment]
Here’s my 2 cents:
- I’m only aware of a single piece of feedback that those icons look clickable. While I can see the point (it makes good design feedback, too) I don’t think there’s enough signal here to chase this single point of feedback at the moment over other better known issues. We could probably use a label like ‘feedback ready to solve’ that we stick on user feedback that has enough signal and is therefore available to pick up and work on, but I don’t really want to decide process in a comment 😄 that probably is a product management concern of deciding what can be worked on and what needs to wait and collect more feedback
- I could see these suggested actions being deprecated for other onboarding CTA patterns, so I also feel less inclined to put effort here
- If I were to put effort here I would not make the icon clickable to match the button. I would make the icon look less clickable, maybe remove the drop shadow and use a more muted tone for the icon that makes this more like a visual decorator than a floating icon-only button (as that was the intent for it to be, purely a visual decorator when we didn’t have time to reach for good, decorating illustrations etc. like we do today — this was designed a long time ago)
- I would be okay calling this a sunk cost, closing the issues, and we all go focus on more valuable things
- Feel free to throw me off the island 🏖️ but please send me to another beautiful one 😎

**Comment by gitstart** [comment]
closing this PR, thanks @ackernaut

**Pull request closed without merging**